h2o-core/src/main/java/water/server/jetty/AbstractHTTPD.java

```diff
@@ -221,31 +211,16 @@ protected void startHttp() throws Exception {
    * @throws Exception
    */
   private void startHttps() throws Exception {
-    _server = makeServer();
+    _server = new Server();
 
     SslContextFactory sslContextFactory = new SslContextFactory(_args.jks);
     sslContextFactory.setKeyStorePassword(_args.jks_pass);
-
-    SslSocketConnector httpsConnector = new SslSocketConnector(sslContextFactory);
-
+    ServerConnector httpsConnector = new ServerConnector(_server, sslContextFactory, makeHttpConnectionFactory());
     if (getIp() != null) {
       httpsConnector.setHost(getIp());
     }
     httpsConnector.setPort(getPort());
-
-    createServer(
-        configureConnector("https", httpsConnector));
-  }
-
-  // Configure connector via properties which we can modify.
-  // Also increase request header size and buffer size from default values
-  // located in org.eclipse.jetty.http.HttpBuffersImpl
-  private Connector configureConnector(String proto, Connector connector) {
-    connector.setRequestHeaderSize(H2O.OptArgs.getSysPropInt(proto+".requestHeaderSize", 32*1024));
```

Comment:
Is there a replacement for these settings? I didn't find anything

Reply:
looks like `Connector` is immutable in Jetty 9. I will search or check with @jakubhava  if he is aware of any replacements.